javascript/Face/sdk_quickstart-single.js.bak

```diff
@@ -0,0 +1,159 @@
+// <snippet_single>
```

Comment:
I'd prefer to rename this file to `sdk_quickstart_cognitiveservices-face.js` to keep the `.js` file extension.